syntax = "proto3";

package window;


service Window {
	rpc MoveWindow(MoveMessage) returns (ResponseMessage) {}
}

message ResponseMessage {
	int32 result = 1;
	bool success = 2;
	string msg = 3;
}

message MoveMessage {
	int32 hwnd = 1;
	int32 left = 2;
	int32 top = 3;
	int32 width = 4;
	int32 height = 5;
	bool repaint = 6;
	string msg = 7;
}